src/__tests__/pages/api/admin/validate.test.ts

```diff
@@ -0,0 +1,116 @@
+/**
+ * @jest-environment node
+ */
+import { PrismaAdapter } from '@next-auth/prisma-adapter';
+import { PrismaClient } from '@prisma/client';
+import { NextApiRequest, NextApiResponse } from 'next';
+import adminValidate from '../../../../pages/api/admin/validate';
+import { mockEmail, mockPassword } from '../../../../__mocks__/pages/api/auth';
+
+const prisma = new PrismaClient();
+const { createUser } = PrismaAdapter(prisma);
+
+beforeEach(async () => {
+  await prisma.user.deleteMany();
+  await prisma.account.deleteMany();
+});
+
+describe('/api/admin/validate: Success', () => {
+  const adminEmail = mockEmail();
+
+  it('Sucessfully validate admin', async () => {
+    const adminUser = await createUser({
```

Comment:
I think this is probably fine for now, but it'd be nice of all these standard tasks like "create admin user with this email" were just functions. Almost like a "test utils" folder. I think it's good in the long run to have the code in tests explicitly be the test itself and not lots of extra set up code that makes files longer than they need to be. Anyway, if this makes sense make a card to do this on Trello and resolve this conversation

Reply:
I still not get this, please explain in the meeting @Opeyem1a